### PR TITLE
Edge install instructions minor updates

### PR DESCRIPTION
**Pull request by deaflynx**
## Pull Request description

Edge instructions updates:
- set tabs position sticky
- code wrapper change background, border
- copy button change color

Before:
Screenshot from 2023-09-15 12-00-06

After:
Screenshot from 2023-09-14 18-00-45

## General checklist

- [ ] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [ ] [Labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-labels) that classify your pull request have been added.
- [ ] The [milestone](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) is specified and corresponds to fix version.  
- [ ] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [ ] Description contains human-readable scope of changes.
- [ ] Description contains brief notes about what needs to be added to the documentation.
- [ ] No merge conflicts, commented blocks of code, code formatting issues.
- [ ] Changes are backward compatible or upgrade script is provided.
- [ ] Similar PR is opened for PE version to simplify merge. Crosslinks between PRs added. Required for internal contributors only.
  
## Front-End feature checklist

- [ ] Screenshots with affected component(s) are added. The best option is to provide 2 screens: before and after changes;
- [ ] If you change the widget or other API, ensure it is backward-compatible or upgrade script is present.
- [ ] Ensure new API is documented [here](https://github.com/thingsboard/thingsboard-ui-help)

## Back-End feature checklist

- [ ] Added corresponding unit and/or integration test(s). Provide written explanation in the PR description if you have failed to add tests.
- [ ] If new dependency was added: the dependency tree is checked for conflicts.
- [ ] If new service was added: the service is marked with corresponding @TbCoreComponent, @TbRuleEngineComponent, @TbTransportComponent, etc.
- [ ] If new REST API was added: the RestClient.java was updated, issue for [Python REST client](https://github.com/thingsboard/thingsboard-python-rest-client) is created.

### CODE REVIEW

File: ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss [hunk 1]
```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 
+@use '@angular/material' as mat;
 @import "../../../../../scss/constants";
+@import '../theme.scss';
 
+$tb-primary: mat.get-color-from-palette($tb-primary);
```
Comment (vvlladd28) [inline ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss:21]: Need to used $tb-primary-color from constants.scss

File: ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss [hunk 2]
```diff
@@ -101,6 +111,12 @@
       padding-right: unset !important;
       padding-left: unset !important;
     }
+    pre[class*="language-"] {
+      border: 1px solid $tb-primary !important;
+    }
+    div {
+      background-color: #F3F6FA;
+    }
```
Comment (vvlladd28) [inline ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss:119]: Why is this style not the _div_ element written above?

File: ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss [hunk 3]
```diff
@@ -63,7 +66,8 @@
 
   .mat-mdc-dialog-content {
     max-height: 80vh;
-    padding: 16px;
+    padding: 0 16px 16px 16px;
+    border-top: 16px solid #fff;
```
[the remark this answers is not in the document]
Comment (deaflynx) [reply ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss:70]: this fixes scrolling empty border bug 
Screenshot from 2023-09-15 16-06-27

**Review by vvlladd28 — APPROVED** [review]

File: ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (vvlladd28) [reply ui-ngx/src/app/modules/home/pages/edge/edge-instructions-dialog.component.scss:67]: If you do not use the sticky position, this problem will not reproduce. If the dimensions of the tab container are correctly set, the header will permanently be fixed.

**Pull request merged**